website/src/components/LeaderBoardItem.tsx

```diff
@@ -0,0 +1,18 @@
+export function RankItem({username,score}){
```

Comment:
For components, I'd like to try and consistently use functional components.  Could you swtich this to

```
const RankItem = ({username, score}) => {
...
}

export default RankItem
```

Overtime I'll be updating the current components to follow that style whenever possible.